kpops/component_handlers/kafka_connect/connect_wrapper.py

```diff
@@ -24,18 +31,12 @@ class ConnectWrapper:
     Wraps Kafka Connect APIs
     """
 
-    def __init__(self, host: str | None):
-        if not host:
-            error_message = (
-                "The Kafka Connect host is not set. Please set the host in the config."
-            )
-            log.error(error_message)
-            raise RuntimeError(error_message)
-        self._host: str = host
+    def __init__(self, config: KpopsConfig) -> None:
```

Comment:
not the entire KpopsConfig, only the Kafka Connect part

Reply:
I did that because we are passing the entire config in [ProxyWrapper](https://github.com/bakdata/kpops/pull/354/files#diff-65902428921d2c088ec1d72082bc05070596e93c5246dc59ff1cd84efcc70539L28).

Reply:
yes, it should also only get passed the relevant Kafka REST config

Reply:
Done 😄